src/ansys/sound/core/psychoacoustics/tonality_din_45681.py

```diff
@@ -0,0 +1,487 @@
+                np.array([]),
+                np.array([]),
+                np.array([]),
+                np.array([]),
+                np.array([]),
+                np.array([]),
+            )
+
+        return (
+            np.array(output[0]),
+            np.array(output[1]),
+            np.array(output[2]),
+            np.array(output[3].data),
+            np.array(output[4].data),
+            np.array(output[5].data),
+            np.array(output[6].data),
+            np.array(output[3].time_freq_support.time_frequencies.data),
+            np.array(output[7]),
+        )
+
+    def get_mean_difference(self) -> float:
+        """Get the DIN 45681 tonality (mean difference DL) in dB.
+
+        Returns
+        -------
+        float
+            Mean difference DL in dB.
+        """
+        return self.get_output()[0]
+
+    def get_uncertainty(self) -> float:
+        """Get the DIN 45681 uncertainty in dB.
+
+        Returns
+        -------
+        float
+            Uncertainty in dB.
+        """
+        return self.get_output()[1]
+
+    def get_tonal_adjustment(self) -> float:
+        """Get the DIN 45681 tonal adjustment in dB.
+
+        Returns
+        -------
+        float
+            Tonal adjustment Kt in dB.
+        """
+        return self.get_output()[2]
+
+    def get_decisive_difference_over_time(self) -> npt.ArrayLike:
+        """Get the DIN 45681 decisive difference DLj in dB over time.
+
+        Returns
+        -------
+        numpy.ndarray
+            Decisive difference DLj in dB over time.
+        """
+        return self.get_output_as_nparray()[3]
+
+    def get_uncertainty_over_time(self) -> npt.ArrayLike:
+        """Get the DIN 45681 decisive difference uncertainty in dB over time.
+
+        Returns
+        -------
+        numpy.ndarray
+            Decisive difference uncertainty in dB over time.
+        """
+        return self.get_output_as_nparray()[4]
+
+    def get_decisive_frequency_over_time(self) -> npt.ArrayLike:
+        """Get the DIN 45681 decisive frequency in Hz over time.
+
+        Returns
+        -------
+        numpy.ndarray
+            Decisive frequency in Hz over time.
+        """
+        return self.get_output_as_nparray()[5]
+
+    def get_tonal_adjustment_over_time(self) -> npt.ArrayLike:
+        """Get the DIN 45681 tonal adjustment Kt in dB over time.
+
+        Returns
+        -------
+        numpy.ndarray
+            Tonal adjustment Kt in dB over time.
+        """
+        return self.get_output_as_nparray()[6]
+
+    def get_time_scale(self) -> npt.ArrayLike:
+        """Get the DIN 45681 time scale in s.
+
+        Returns
+        -------
+        numpy.ndarray
+            Computation timestamps in s.
+        """
+        return self.get_output_as_nparray()[7]
+
+    def get_spectrum_number(self) -> int:
+        """Get the number of spectra.
+
+        Returns the number of analyzed spectra, ie the number of computation time steps.
```

Comment:
we are encouraged not to use latin abbreviations in documentation (not universally understood). 'that is' is better.